extensions/amp-form/0.1/test/test-amp-form.js

```diff
@@ -2322,12 +2327,10 @@ describes.repeated('', {
         expect(ampForm.urlReplacement_.expandInputValueSync)
             .to.have.been.calledWith(canonicalUrlField);
 
-        return whenCalled(form.submit).then(() => {
-          expect(form.submit).to.have.been.calledOnce;
-          expect(clientIdField.value).to.equal('');
-          expect(canonicalUrlField.value).to.equal(
-              'https%3A%2F%2Fexample.com%2Famps.html');
-        });
+        expect(form.submit).to.have.been.called;
```

Comment:
Can you help me understand why you replaced `calledOnce` with `called`? We would want to know if the form was submitted multiple times and avoid it.

Reply:
Oops, didn't know I did that. Thanks! 😄